core/src/main/java/org/ethereum/beacon/core/Hashable.java

```diff
@@ -3,12 +3,4 @@
 import tech.pegasys.artemis.ethereum.core.Hash;
 
 /** An interface of a hashable entity. */
-public interface Hashable<H extends Hash> {
-
-  /**
-   * Returns a hash of the object.
-   *
-   * @return a hash.
-   */
-  H getHash();
-}
+public interface Hashable<H extends Hash> {}
```

Comment:
Why do we need it now?

Reply:
I left it as a part of type safety but there is not much benefit in that.